edge_orchestrator/edge_orchestrator/domain/use_cases/supervisor.py

```diff
@@ -174,6 +174,8 @@ def apply_business_rules(self, item: Item) -> str:
                     'parameters']  # noqa
 
                 last_model_inferences = get_last_inference_by_camera(item.inferences[camera_id])
+                if last_model_inferences == 'NO_DECISION':
```

Comment:
on peut utiliser l'enum pour ça ?

ça donnerait un truc comme ça  
```suggestion
                if last_model_inferences == Decision.NO_DECISION:
```